tests/testthat/test_real.R

```diff
@@ -283,6 +282,32 @@ test_that("variableVectorProbabilities", {
   expect_true(is_real_vector(v, allow_na_values = TRUE))
 })
 
+test_that("variableVectorProbabilitiesMinMax", {
+
+  v <- c(0.2, 0.0, 1.0)
+  v2 <- c(1.2, -1.3, 3.0)
+
+  expect_true(is_probability_vector(v))
+  expect_false(is_probability_vector(v2))
+
+  err <- tryCatch({
```

Comment:
I think you can use expect_error now. The noisy form you see in qscheck is because back when I started there was no expect_error in testthat.